Client/Frontend/Home/FxPocketHomeHorizontalCell.swift

```diff
@@ -0,0 +1,186 @@
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .subheadline,
+                                                                   maxSize: FxHomeHorizontalCellUX.titleFontSize)
+        label.numberOfLines = 2
+    }
+
+    private lazy var sponsoredStack: UIStackView = {
+        let stack = UIStackView(arrangedSubviews: [sponsoredIcon, sponsoredLabel])
+        stack.axis = .horizontal
+        stack.spacing = 8
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        return stack
+    }()
+
+    private lazy var sponsoredLabel: UILabel = {
+        let label = UILabel()
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .subheadline,
+                                                                       maxSize: FxHomeHorizontalCellUX.sponsoredFontSize)
+        label.textColor = .secondaryLabel
+        label.numberOfLines = 1
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var sponsoredIcon: UIImageView = {
+        let image = UIImageView(image: UIImage(named: "sponsored-star"))
+        NSLayoutConstraint.activate([
+            image.heightAnchor.constraint(equalToConstant: 12),
+            image.widthAnchor.constraint(equalToConstant: 12)
+        ])
+        image.translatesAutoresizingMaskIntoConstraints = false
+        return image
+    }()
+
+    private let descriptionLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .caption1,
+                                                                   maxSize: FxHomeHorizontalCellUX.siteFontSize)
+        label.textColor = .label
+    }
+
+    // MARK: - Variables
+    var notificationCenter: NotificationCenter = NotificationCenter.default
+
+    // MARK: - Inits
+
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+
+        applyTheme()
+        setupNotifications(forObserver: self,
+                           observing: [.DisplayThemeChanged])
+        setupLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    deinit {
+        notificationCenter.removeObserver(self)
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        heroImageView.image = nil
+        descriptionLabel.text = nil
+        titleLabel.text = nil
+        applyTheme()
+    }
+
+    // MARK: - Helpers
+
+    func configure(viewModel: FxPocketHomeHorizontalCellViewModel) {
+        titleLabel.text = viewModel.title
+        descriptionLabel.text = viewModel.description
+
+        heroImageView.sd_setImage(with: viewModel.imageURL)
+        sponsoredLabel.text = "Sponsored"
```

Comment:
Seems this string "Sponsored" should be translated?

Reply:
Updated.